common/script_values/wc_men_at_arms_values.txt

```diff
@@ -0,0 +1,139 @@
+﻿######################
+# Men-at-Arms Values #
+######################
+
+#########
+# Costs #
+#########
+
+@recr_to_maint_factor = 0.004
+@high_maint_mult = 3
+
+###Vanilla Refrence for Camels/Elekks
+@light_cavalry_recruitment_cost = 85
+@light_cavalry_low_maint_cost = 0.35
+@light_cavalry_high_maint_cost = @[light_cavalry_low_maint_cost * high_maint_mult]
+
+@heavy_cavalry_recruitment_cost = 200
+@heavy_cavalry_low_maint_cost = 0.7
+@heavy_cavalry_high_maint_cost = @[heavy_cavalry_low_maint_cost * high_maint_mult]
+
+### Base Cost Values
+@massive_infantry_recruitment_cost = 150
+@massive_infantry_low_maint_cost = @[massive_infantry_recruitment_cost * recr_to_maint_factor]
+@massive_infantry_high_maint_cost = @[massive_infantry_low_maint_cost * high_maint_mult]
+
+@caster_recruitment_cost = 100
+@caster_low_maint_cost = @[caster_recruitment_cost * recr_to_maint_factor]
+@caster_high_maint_cost = @[caster_low_maint_cost * high_maint_mult]
+
+@light_aerial_recruitment_cost = 100
+@light_aerial_low_maint_cost = @[light_aerial_recruitment_cost * recr_to_maint_factor]
+@light_aerial_high_maint_cost = @[light_aerial_low_maint_cost * high_maint_mult]
+
+@massive_aerial_recruitment_cost = 300
+@massive_aerial_low_maint_cost = @[massive_aerial_recruitment_cost * recr_to_maint_factor]
+@massive_aerial_high_maint_cost = @[massive_aerial_low_maint_cost * high_maint_mult]
+
+### Base MAA
+massive_infantry_recruitment_cost = @[massive_infantry_recruitment_cost]
+massive_infantry_low_maint_cost = @[massive_infantry_low_maint_cost]
+massive_infantry_high_maint_cost = @[massive_infantry_high_maint_cost]
+
+caster_recruitment_cost = @[caster_recruitment_cost]
+caster_low_maint_cost = @[caster_low_maint_cost]
+caster_high_maint_cost = @[caster_high_maint_cost]
+
+light_aerial_recruitment_cost = @[light_aerial_recruitment_cost]
+light_aerial_low_maint_cost = @[light_aerial_low_maint_cost]
+light_aerial_high_maint_cost = @[light_aerial_high_maint_cost]
+
+massive_aerial_recruitment_cost = @[massive_aerial_recruitment_cost]
+massive_aerial_low_maint_cost = @[massive_aerial_low_maint_cost]
+massive_aerial_high_maint_cost = @[massive_aerial_high_maint_cost]
+
+### Cultural MAA
+#Add custom costs for each cultural maa here
+### Regional MAA
+#Add custom costs for each regional maa here
+
+##############
+# AI Weights #
+##############
+#Which cultures like what type of unit
+# Warcraft
+culture_ai_weight_caster = {
+	value = 125
```

Comment:
Explain the formula.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107890870-b88a8380-6f34-11eb-9501-5c4ea2daaa44.png)

</details>
Why AAIW is taken into consideration? AAIW is the cost of modifiers not included in the Paradox's AI picking formula. Thus these modifiers are taken in your formula twice! First, in Total Cost, second, in AAIW.